edb/lang/graphql/types.py

```diff
@@ -468,8 +474,11 @@ def __init__(self, schema, **kwargs):
         if edb_base is None and self.edb_type:
             edb_base = schema.edb_schema.get(self.edb_type)
 
-        assert edb_base is not None
-        assert schema is not None
+        if edb_base is None:
+            raise GraphQLTranslationError(
+                "cannot determine the EdgeDB base type")
+        if schema is None:
+            raise GraphQLTranslationError("schema is missing")
```

Comment:
You use `schema` a few lines above, this check is pointless.

Reply:
`schema.get()` without a `default=` raises a `SchemaError`.